doc/en/example/nonpython.rst

```diff
@@ -84,8 +84,9 @@ interesting to just look at the collection tree::
     platform linux -- Python 3.x.y, pytest-3.x.y, py-1.x.y, pluggy-0.x.y
     rootdir: $REGENDOC_TMPDIR/nonpython, inifile:
     collected 2 items
-    <YamlFile 'test_simple.yml'>
-      <YamlItem 'hello'>
-      <YamlItem 'ok'>
+    <Package '$REGENDOC_TMPDIR/nonpython'>
```

Comment:
this one is suspicious

Reply:
@RonnyPfannschmidt I believe it is the fix from #3798, because now it is collecting packages again.

cc @jonozzz

Reply:
(I've spoken with @RonnyPfannschmidt on chat and explained the issue with him)